shell/assets/translations/en-us.yaml

```diff
@@ -1313,6 +1313,8 @@ cluster:
         macAddress: Mac Address
         macFormat: 'Invalid MAC address format.'
       vGpus:
+        warnings:
+          minimumAllocatable: It's highly recommended to select a vGPU with a number of allocatable devices greater then the number of nodes (Machine Count) to avoid "un-schedulable" errors after cluster updates.
         errors:
           notAllocatable: '"VGPUs" not allocatable. There are not enough [{vGpus}] devices to be allocated to each node in machine pool [{pool}]'
```

Comment:
Anywhere we have vGPU, we should standardize the spelling as such. I think the screenshot shows `vGpu` which we have to be careful with for copy but variable names are OK.

Reply:
Typo: "then" should also be *than*